tests/virt_autotest/libvirt_extend_storage_lvm.pm

```diff
@@ -104,6 +105,24 @@ sub wipe_hard_disk {
     assert_script_run("dd if=/dev/zero of=$hard_disk_name count=1M", timeout => 1500, fail_message => "Failed to wipe hard disk clean on $hard_disk_name");
 }
 
+# Reomve volume group associated with a disk partition
+sub remove_volume_group {
+    my %args = @_;
+    $args{disk} //= '';
+    $args{vg} //= '';
+    die("Neither disk nor vg should be empty") unless ($args{disk} and $args{vg});
+
+    my $timeout = 180;
+    record_info("Reomve vg $args{vg} forcibly");
+    script_run("vgremove -f -y $args{vg}", timeout => $timeout);
+    script_run("vgdisplay $args{vg}", timeout => $timeout);
+
+    record_info("Remove physical volume $args{vg} forcibly");
+    script_run("pvremove -f -y $args{disk}", timeout => $timeout);
+    script_run("pvdisplay", timeout => $timeout);
```

Comment:
Nice PR! 
Just a minor comment. I guess you use script_run because the vg and pv may not always be available? Then how about after each set of removing operations, adding some assert check for no existence any more?

Reply:
Done.